IPsec documentation updated. (#30822)
cilium-team/cilium: Documentation/security/network/encryption-ipsec.rst

Key Rotation
============

> Starting from Cilium CLI v0.15.23 IPsec keys can be rotated using the following command:

Reviewer: ```suggestion
Starting from Cilium CLI v0.15.23, you can rotate the IPsec keys using the following command:
```

Author: Fixed, thanks!

.. code-block:: shell-session

    cilium encryption rotate-key

> Alternatively, the following commands can be used to obtain the same result:

Reviewer: ```suggestion
Alternatively, you can use the following commands to obtain the same result:
```

Author: Fixed, thanks!

allow for the new keys to be deployed and applied across all clusters,
which you can do with the agent flag ``ipsec-key-rotation-duration``.

Encryption status
=================

> Starting from Cilium CLI v0.15.23 cluster wide encryption status and IPsec key can be displayed using the following commands:

Reviewer: This read like the key itself can be displayed, is it the case?

```suggestion
Starting from Cilium CLI v0.15.23, you can display cluster-wide encryption status and the status of the IPsec key with the following commands:
```

Author: Fixed, thanks!
